src/main.ts

```diff
@@ -0,0 +1,29 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+import { NestExpressApplication } from '@nestjs/platform-express';
+import { logger } from './shared/logger/winston.config';
+import { LoggingValidationPipe } from './shared/pipes/logging-validation.pipe';
+
+/**
+ * Initializes and starts the application.
+ * @returns {Promise<void>} A promise that resolves when the application is running.
+ */
+async function bootstrap() {
+  const app = await NestFactory.create<NestExpressApplication>(AppModule);
+  app.useGlobalPipes(new LoggingValidationPipe());
+  app.useBodyParser('json', {
+    type: ['application/json', 'application/cloudevents+json'],
+  });
+  app.enableCors({
```

Comment:
ohm, what? why?

Reply:
I had CORS issues with the experiment-config-frontend without this

Reply:
but that's not really a long-term solution, right?
because for example with k8s, you cannot guarantee that the frontend is available at this address
the usual approach would be using a reverse proxy, the question is, do we want to make the experiment config stuff also available via our regular frontend? maybe similar how the GraphiQL is handled (only for admin users)?
of course we must also support to use the experiment config without a token (when directly accessing it) to make itegrating it with load testing stuff reasonably possible, so the auth check would need to be done in some kind of proxy, or we set a particular header in the proxy which requires auth checking
idk, just writing down some ideas here...

Reply:
I would ignore any auth checks for the moment.
Integrating it into the frontend would be an idea but I wouldnt prioritise it.
I will try to make it work without the CORS

Reply:
Worst case I will simply allow all origins here. Then we do not have an kubernetes issue.
This part of the system also does not need to be secure IMO, since its inside of the experimenting side

Reply:
when you do the nginx stuff in the frontend, this will hopefully no longer be required

Reply:
CORS is removed